server/controllers/ticket/search.php

```diff
@@ -167,11 +167,17 @@ public function getSQLQueryWithOrder($inputs, $query) {
     private function setQueryFilters($inputs, &$filters){
         if(array_key_exists('tags',$inputs)) $this->setTagFilter($inputs['tags'], $filters);
         if(array_key_exists('closed',$inputs)) $this->setClosedFilter($inputs['closed'], $filters);
-        if(array_key_exists('assigned',$inputs)) $this->setAssignedFilter($inputs['assigned'], $filters);
-        if(array_key_exists('unreadStaff',$inputs)) $this->setSeenFilter($inputs['unreadStaff'], $filters);
+        if(array_key_exists('assigned',$inputs)) {
+            $this->setAssignedFilter($inputs['assigned'], $filters);
+        }
+        if(array_key_exists('unreadStaff',$inputs)) {
+            $this->setSeenFilter($inputs['unreadStaff'], $filters);
+        }
         if(array_key_exists('dateRange',$inputs)) $this->setDateFilter($inputs['dateRange'], $filters);
         if(array_key_exists('departments',$inputs) && array_key_exists('allowedDepartments',$inputs) && array_key_exists('staffId',$inputs)){
-            if(!$this->ignoreDeparmentFilter) $this->setDepartmentFilter($inputs['departments'],$inputs['allowedDepartments'], $inputs['staffId'], $filters);  
+            if(!$this->ignoreDeparmentFilter){
+                $this->setDepartmentFilter($inputs['departments'],$inputs['allowedDepartments'], $inputs['staffId'], $filters);
+            }
```

Comment:
Since these lines didn't really change, let's leave them in their original state.